functions-methods.js

```diff
@@ -33,4 +63,21 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(string3) {
+    const email = string3;
+    if (email.includes("@") && email.charAt(string3.length-1) !== "." && email.includes(",") === false) {
+        const emailAdress = true;
+        return emailAdress;
+    } else {
+        const emailAdress = false;
+        return emailAdress;
+    }
```

Comment:
Dit kun je nog wat versimpelen
```suggestion
    return email.includes("@") && email.charAt(string3.length-1) !== "." && !email.includes(",");
```